reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -16,6 +22,20 @@
   z-index: 2;
   background-color: white;
   color: #133a53;
+  
+}
+
+.darkmode table th {
```

Comment:
selector should be .darkmode .heatmap table th

Reply:
✅